ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationPeriodValidationIT.scala

```diff
@@ -69,6 +115,116 @@ class CommandDeduplicationPeriodValidationIT extends LedgerTestSuite {
           )
         ),
       )
-    }
+      metadataLongestDuration = extractErrorInfoMetadataValue(failure, "longest_duration")
+      // request accepted and the metadata value is valid
+      _ <- ledger.submit(
```

Comment:
Don't we have to do extra assertions on the result?

Reply:
No, it's just about not failing